### ISSUE TITLE
penalty_ccp_solve has issues with sol.table()

### ISSUE THREAD

**Issue by 1ozturkbe** [convexengineering/gpkit#1441]
I used it to solve SPaircraft, and it returned the following error:
```python
In [19]: print(sol.table())
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-19-7879b4d2ee95> in <module>()
----> 1 print(sol.table())

c:\users\berk\dropbox (mit)\mit graduate school\code\gpkit\gpkit\solution_array.pyc in table(self, showvars, tables, **kwargs)
    641                 strs += [""]
    642             elif table in TABLEFNS:
--> 643                 strs += TABLEFNS[table](self, showvars, **kwargs)
    644             elif table in self:
    645                 data = self[table]

c:\users\berk\dropbox (mit)\mit graduate school\code\gpkit\gpkit\solution_array.pyc in tight_table(self, _, ntightconstrs, tight_senss, **kwargs)
     73     data = [((-float("%+6.2g" % c.relax_sensitivity), str(c)),
     74              "%+6.2g" % c.relax_sensitivity, id(c), c)
---> 75             for c in self.model.flat() if c.relax_sensitivity >= tight_senss]
     76     if not data:
     77         lines = ["No constraints had a sensitivity above %+5.1g."

AttributeError: 'SignomialInequality' object has no attribute 'relax_sensitivity'
```
Will dig in and report here.

**Comment by 1ozturkbe** [comment]
Oh, it seems relax_sensitivity should exist for the last GP and not the SP, so something wrong with solution withdrawal.

**Comment by bqpd** [comment]
it's probably a missing `sgp_parent`, it doesn't happen on simpler models.

**Comment by bqpd** [comment]
@1ozturkbe I can't seem to reproduce this with SPAircraft.py, could you make an MWE?

**Comment by bqpd** [comment]
Nevermind, got it!

**Comment by 1ozturkbe** [comment]
Sweet, looking forward to seeing what the issue actually is!

**Comment by bqpd** [comment]
It's in the way relaxed creates new constraints that have no connection to their originals.

**Comment by bqpd** [comment]
Check the last commit in #1443

**Comment by 1ozturkbe** [comment]
Thanks for the heads up, just reviewed! relaxed.sgp_parent = self!

**Comment by 1ozturkbe** [comment]
This was fixed in #1443.
